socket.c

```diff
@@ -40,166 +40,179 @@ extern int debug;
 /*
  * gethostbyname() wrapper. Return 1 if OK, otherwise 0.
```

Comment:
```suggestion
 * Resolve hostname. Return 1 if OK, otherwise 0.
```
Some better description would also be ok :)